compose/galaxy-configurator/run.sh

```diff
@@ -168,10 +171,19 @@ for conf in "${galaxy_configs[@]}"; do
   mv -f "/tmp/$conf" "${GALAXY_CONF_DIR}/$conf"
 done
 
+# TODO: Install tools into Galaxy (Singularity)
+galaxy_tools_file="tools.yaml"
+echo "Installing galaxy tools"
+python -m workbench download-jar
```

Comment:
looks good... this gets run each time the system restarts, right? might want to make sure it is caching and not re-downloading / re-installing unncessarily.

then, one extra step we need - but it will need it own config - is installing tools that aren't in the workflows. most importantly this is data manager. finally DMs need to be run at the appropriate point in the config.